core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -480,19 +482,11 @@ oppia.factory('ExplorationPlayerService', [
       // user is not logged in or has not uploaded a profile picture, or the
       // player is in preview mode.
       getUserProfileImage: function() {
```

Comment:
Maybe should add Async to end of this name too?

Reply:
Done.